log4j-core/src/main/java/org/apache/logging/log4j/core/util/SystemClock.java

```diff
@@ -29,4 +34,13 @@ public final class SystemClock implements Clock {
     public long currentTimeMillis() {
         return System.currentTimeMillis();
     }
+
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
Personally I find `{@inheritDoc}` a little bit useless, since IDEs will look at the Javadoc of the parent class/implemented interface anyway, if the Javadoc is missing.

Reply:
Sounds good, will remove this too!